### ISSUE TITLE
Expose card brand in `/v1/sub/customer/{uid}` resource

### ISSUE THREAD

**Issue by lmorchard** [mozilla/subhub#377]
Over in the UI on FxA, [we have a few spots where we'd like to display a credit card icon](https://github.com/mozilla/fxa/issues/3037). But, the card brand does not seem to be exposed in responses. Can we get that brand added to the response? (I *think* that response [gets constructed hereabouts](https://github.com/mozilla/subhub/blob/master/src/sub/payments.py#L485).)

**Comment by stlhood** [comment]
In what timeframe is this needed?

**Comment by lmorchard** [comment]
This is actually needed by the tiered subscription UX. I think we overlooked it while reviewing the mocks - I just now realized it while working on implementation of the UI. We've so far punted on including the card icon in other spots. We could decide to do that again - but if it's a hard requirement, then we need this data to render that icon.

<img width="275" alt="Capture" src="https://user-images.githubusercontent.com/21687/68795545-0d3ea080-0606-11ea-9b64-c751e05285ed.PNG">
